packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -150,7 +150,7 @@ class Manuscript extends BaseModel {
     })
     // todo why do I need to do this?
     await Promise.all(
```

Comment:
Is this another instances of the wrapping in a `Promise.all` for no real reason?

Reply:
the reason is that if we used a forEach, then it wouldn't wait until all the $loadRelated calls are done

https://medium.com/@pyapar/another-approach-using-map-instead-of-foreach-82a85f536c9c

Reply:
Maybe let's get rid of the comment then?

Reply:
@erezmus ... can you remove this comment- as I think its misleading if you understand  it now :)